src/gis/settingup_background_map.md

```diff
@@ -110,6 +130,37 @@ Note that instructions below require familiarity with the terminal. In addition,
   - Re-run `./quickstart.sh my`
   - The above process should produce an MBTiles for your clipped OSM file
   - You can load the file and style it using one of the <GitHubRepo id="openmaptiles/osm-bright-gl-style" desc="OpenMapTiles styles" />
+:::
+
+
+#### Downloading Vector Tiles Using Mergin Maps Plugin for QGIS 
```

Comment:
I suggest to add somewhere in this article massive warning to users that they are legally responsible for usage or downloads of any maps and their usage in their projects, so they always need to consult EULA or T&C of any maps provider for attibution and their usage